src/main/java/org/mafagafogigante/dungeon/schema/JsonRule.java

```diff
@@ -0,0 +1,14 @@
+package org.mafagafogigante.dungeon.schema;
+
+import com.eclipsesource.json.JsonValue;
+
+/**
+ * General interface for json value validation rules.
+ */
+public interface JsonRule {
+
+  /**
+   * Key method for json validation.
```

Comment:
This comment does not mean anything. "Validates the provided JSON value." would be much better.